mandatory/1-fix-functions.js

```diff
@@ -11,8 +11,7 @@
 */
 
 function mood() {
```

Comment:
I think I just found a mistake in the homework 😅 , Line 13 should read `function mood(isHappy)`.

The answer is to remove line 14 as you have done, line 15 shouldn't be necessary. Sorry about the mistake in the homework.

Reply:
I see, Thank you for letting me know, I've change the code